docs/API.md

```
@@ -522,6 +532,107 @@ combined input the live flow uses.
 
 ---
 
+### 12. Get Current Token
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)